Views/Home/Index.cshtml

```diff
@@ -64,34 +62,124 @@
+                        </div>
+                    </div>
+                </div>
+            </a>
         </div>
     </div>
 
-    <!-- Content Row -->
     <div class="row">
+        <div class="col-xl-4 mb-4">
+            <a class="card lift h-100" href="#!">
+                <div class="card-body d-flex justify-content-center flex-column">
+                    <div class="d-flex align-items-center justify-content-between">
+                        <div class="mr-3">
+
+                            <i class="feather-xl text-primary mb-3" data-feather="bar-chart-2"></i>
+                            <h5> Visualizer </h5>
+                            <div class="text-muted small">Create informative visual view of your data.</div>
+                        </div>
+                    </div>
+                </div>
+            </a>
+        </div>
+        <div class="col-xl-4 mb-4">
+            <a class="card lift h-100" asp-controller="Triggers" asp-action="Index">
+                <div class="card-body d-flex justify-content-center flex-column">
+                    <div class="d-flex align-items-center justify-content-between">
+                        <div class="mr-3">
+                            <i class="feather-xl text-danger mb-3" data-feather="cloud-lightning"></i>
+                            <h5>Triggers</h5>
+                            <div class="text-muted small">Manage triggers for webhook communication</div>
+                        </div>
+                    </div>
+                </div>
+            </a>
+        </div>
+        <div class="col-xl-4 mb-4">
+            <a class="card lift h-100" asp-controller="Forecast" asp-action="Index">
+                <div class="card-body d-flex justify-content-center flex-column">
+                    <div class="d-flex align-items-center justify-content-between">
+                        <div class="mr-3">
 
-        <!-- Content Column -->
-        <div class="col-lg-6 mb-4">
+                            <i class="feather-xl text-success mb-3" data-feather="fast-forward"></i>
+
+                            <h5>Forecast</h5>
+                            <div class="text-muted small">Predict next event using gathered data.</div>
+                        </div>
+                    </div>
+                </div>
+            </a>
+        </div>
+    </div>
 
+    <div class="row">
+        <div class="col-xxl-8 col-xl-6 col-lg-6 col-12">
+            <div class="card mb-4">
+                <div class="card-header border-bottom">
+                    Overview
+                </div>
+                <div class="card-body">
+                    No overview provided...
+                </div>
+            </div>
 
+        </div>
 
+        <div class="col-xxl-4 col-xl-6 col-lg-6 col-12">
+            <div class="card mb-4">
+                <div class="card-header">Venjix Information</div>
+                <div class="card-body">
+                    Version : 0.0.0.0

```

Comment:
Version should be retrieved from static field from code, maybe using Reflection or another static field so this value will not be hardcoded.

Reply:
For now I'll add the version info to DashboardModel.